paper_sections/05-computational_and_statistical_issues.Rmd

```diff
@@ -628,9 +636,21 @@ to balance penalties between nested smooth terms (this is why nested random
 effects can be fitted). We have observed that **mgcv** still occasionally
 finds solutions with simulated data where the global term is over-smoothed.
 
-To avoid this issue, we recommend both careful choice of basis and setting model degrees of freedom so that groupwise terms are either slightly less flexible than the global term or have a smaller null space. In the examples in section III, we used smoothers with an unpenalized null space (standard thin plate splines) for the global smoother and ones with no null space for the groupwise terms[^gsnull]. When using thin plate splines, it may also help to use splines with a lower order of derivative penalized in the groupwise smoothers than the global smoothers, as lower-order "tp" splines have fewer basis functions in the null space. For example, we used `m=2` (penalizing squared second derivatives) for the global smoother, and `m=1` (penalizing squared first derivatives) for groupwise smoothers in models *GS* and *GI*. Another option is to use a lower number of basis functions (`k`) for groupwise relative to global terms.  This will reduce the maximum flexibility possible in the groupwise terms. We do caution that these are just rules of thumb. In cases where an accurately estimated global smoother is essential, we recommend either fitting model *G* or using specialized functional regression software such as the `fosr` function in the *refund* package [@scheipl_functional_2014], which allows the user to enforce constraints on the groupwise smoothers so that they always sum to zero at any given point (avoiding the collinearity issue). Also, see below for more information on functional regression.
+To avoid this issue, we recommend both careful choice of basis and setting model degrees of freedom so that groupwise terms are either slightly less flexible than the global term or have a smaller null space. In the examples in section III, we used smoothers with an unpenalized null space for the global smoother and ones with no null space for the groupwise terms[^gsnull]. When using thin plate splines, this can be done with splines with a lower order of derivative penalized in the groupwise smoothers than the global smoothers, as lower-order "tp" splines have fewer basis functions in the null space. For example, we used `m=2` (penalizing squared second derivatives) for the global smoother, and `m=1` (penalizing squared first derivatives) for groupwise smoothers in models *GS* and *GI*. Another option is to use a lower number of basis functions (`k`) for groupwise relative to global terms.  This will reduce the maximum flexibility possible in the groupwise terms. We do caution that these are just rules of thumb. 
+
+As noted above, interpreting the shape of global terms and group-wise deviations seperately for  *GS* models fit using tensor-product group-level terms is complicated by the fact that **mgcv** will drop some basis-functions for the group-level term to prevent perfect concurvity. 
```

Comment:
> will drop some basis-functions for the group-level term to prevent perfect concurvity.

Should this be: "will drop some basis-functions *from* the groiup-level *terms* to prevent perfect concurvity."?